processing/src/main/java/io/druid/query/aggregation/first/DoubleFirstAggregatorFactory.java

```diff
@@ -116,10 +130,10 @@ public AggregatorFactory getCombiningFactory()
     return new DoubleFirstAggregatorFactory(name, name)
     {
       @Override
-      public Aggregator factorize(ColumnSelectorFactory metricFactory)
+      public Pair<Aggregator, BaseNullableColumnValueSelector> factorize2(ColumnSelectorFactory metricFactory)
       {
-        final BaseObjectColumnValueSelector selector = metricFactory.makeColumnValueSelector(name);
-        return new DoubleFirstAggregator(null, null)
+        final ColumnValueSelector selector = metricFactory.makeColumnValueSelector(name);
+        return Pair.of(new DoubleFirstAggregator(null, null)
```

Comment:
Please reformat to break line after `of(`

Reply:
done.